app/models/searchgov_domain.rb

```diff
@@ -24,7 +24,7 @@ class DomainError < StandardError; end
   attr_reader :response
 
   scope :ok, -> { where(status: OK_STATUS) }
-  scope :not_ok, -> { where.not(status: OK_STATUS) }
+  scope :not_ok, -> { where.not(status: OK_STATUS).or(where(status: nil)) }
```

Comment:
Please add a spec to verify that this scope will include a record with `status: nil`. Be sure to practice TDD to ensure that we have sufficient specs.